tcms/testplans/templates/testplans/get.html

```diff
@@ -0,0 +1,81 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+{% load comments %}
+{% load extra_filters %}
+{% load attachments_tags %}
+
+{% block title %}TP-{{ object.pk }}: {{ object.name }}{% endblock %}
+{% block body_class %}cards-pf{% endblock %}
+
+{% block contents %}
+    <div class="container-cards-pf">
+    <!-- Important:  if you need to nest additional .row within a .row.row-cards-pf, do *not* use .row-cards-pf on the nested .row  -->
+    <h1 class="col-md-12" style="margin-top: 0">
+        <span id="test_plan_pk"
+              data-testplan-pk="{{ object.pk }}"
+        >TP-{{ object.pk }}:</span> {{ object.name }}
+    </h1>
+    <div class="row row-cards-pf">
+        <div class="col-xs-12 col-sm-12 col-md-9">
+            <div class="card-pf card-pf-accented">
+                <div class="card-pf-body">
+                    <div class="markdown-text">
+                        {{ object.text|markdown2html }}
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="col-xs-12 col-sm-12 col-md-3">
+            <div class="card-pf card-pf-accented card-pf-aggregate-status" style="text-align: left">
+                <h2 class="card-pf-title">
+                    <span class="fa pficon-user"></span>{% trans 'Author' %}:
+                    <a href="{% url "tcms-profile" object.author.username %}">{{ object.author.username }}</a>
+                </h2>
+
+                <h2 class="card-pf-title">
+                    <span class="fa fa-calendar"></span>{{ object.create_date }}
+                </h2>
+
+                <h2 class="card-pf-title">
+                <span id="product_pk"
+                      data-product-pk="{{ object.category.product.pk }}"
```

Comment:
TestPlan doesn't have category, it does have Product. Not sure why this works, probably this is an auto-generated related objects manager.

Reply:
It doesnt work "<span id="product_pk" **data-product-pk=""** class="fa fa-shopping-cart"></span>", I will remove the data attribute and if add it later on if I needed it.